drivers/spi/spi_ll_stm32.h

```diff
@@ -23,4 +23,59 @@ struct spi_stm32_data {
 	struct spi_context ctx;
 };
 
+static inline u32_t spi_stm32_ll_func_IsActiveFlag_TXE(SPI_TypeDef *spi)
```

Comment:
Are these function names sufficiently abstracted?
Also I think you can spare the `spi_stm32` to minimize function names length.

Reply:
I see what you mean, we could take the opportunity to do "is_rx_empty" "is_tx_not_empty" "is_busy"... On my way !